exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -49,6 +49,7 @@
   String USER_RPC_TIMEOUT = "drill.exec.rpc.user.timeout";
   String METRICS_CONTEXT_NAME = "drill.exec.metrics.context";
   String USE_IP_ADDRESS = "drill.exec.rpc.use.ip";
+  String BIT_ADVERTISED_HOST = "drill.exec.rpc.bit.advertised.host";
```

Comment:
advertised.host -> advertised-host (or advertised_host). Here, "advertised" does not seem to form a group, so no need for dot.